documentation/manual/configuration.textile

```diff
@@ -745,11 +745,11 @@ h3(#java.source). java.source
 
 Java source level, which overrides the @java.version@ system property. For example:
 
-bc. java.source=1.8
+bc. java.source=17
 
-Values:  @1.7@ (No longer supported since 1.5.0), @1.8@, @9@, @10@, @11@, @12@, @13@, @14@, @15@, @17@.
+Values:  @1.7@ (No longer supported since 1.5.0), @1.8@ (No longer supported since 1.6.x), @9@ (No longer supported since 1.6.x), @10@ (No longer supported since 1.6.x), @11@, @12@, @13@, @14@, @15@, @17@.
```

Comment:
I think that it should be 'No longer supported since 1.7.0' has the 1.6.0 still support java 8 ?

Reply:
Only if it will be next release. Because code already is uncompilable with java 8.